src/identity/contactMapping.ts

```diff
@@ -243,6 +275,43 @@ function* fetchWalletAddresses(e164Number: string) {
   }
 }
 
+function* fetchAddressVerification(address: string) {
+  try {
+    const walletAddress = yield* select(walletAddressSelector)
+    const signedMessage = yield* call(retrieveSignedMessage)
+
+    const addressVerificationQueryParams = new URLSearchParams({
+      address,
+      clientPlatform: Platform.OS,
+      clientVersion: DeviceInfo.getVersion(),
+    }).toString()
+
+    const response: Response = yield* call(
+      fetch,
+      `${networkConfig.checkAddressVerifiedUrl}?${addressVerificationQueryParams}`,
+      {
+        method: 'GET',
+        headers: {
+          'Content-Type': 'application/json',
+          authorization: `Valora ${walletAddress}:${signedMessage}`,
+        },
+      }
+    )
+
+    if (!response.ok) {
+      throw new Error(
+        `Failed to look up address verification: ${response.status} ${response.statusText}`
+      )
+    }
+
+    const { data }: { data: { addressVerified: boolean } } = yield* call([response, 'json'])
+    return data.addressVerified
+  } catch (error) {
+    Logger.debug(`${TAG}/fetchAddressVerification`, 'Unable to look up address', error)
```

Comment:
same log question as above, probably this one is more useful than the above one since this catch swallows the error and throws a generic message

Reply:
I'll turn this one into a warn, and leave the other as debug.